src/core.js

```diff
@@ -179,6 +179,26 @@ export default class Core {
       _setDataset(el, attribute)
   }
 
+  ajax(path, params = {}, options = {}) {
```

Comment:
how will look a call that requires options, but doesn't need params?

Reply:
`ajax(path, {}, { credentials: 'include' })`

Reply:
Maybe better add params to options?

Reply:
Not sure, now that we have another param, I think this method signature is too large. Maybe we can just use something like: `ajax(path, { params = {}, options = {} })` and all custom options go inside the `options` object. Another approach: keep params separated and make options to handle all fetch options + extras.

**TL;DR**
1) ajax(path, params, options, dataType)
2) ajax(path, { params, options })
3) ajax(path, params, options = {})